docs/source/mnn/index.rst

```diff
@@ -0,0 +1,29 @@
+sherpa-mnn
+===========
+
```

Comment:
Please add this documentat to
https://github.com/k2-fsa/sherpa/blob/master/docs/source/index.rst